guardduty_slack/aws_s3_bucket.tf

```diff
@@ -1,5 +1,6 @@
 resource "aws_s3_bucket" "main" {
   bucket_prefix = "${var.s3_bucket_name}-"
+  tags          = var.tags
```

Comment:
[minor suggestion] The code alignments broke consistency in the source code formatting in this repo.
It needs consistency.

Reply:
Thank you for your review comment. I fixed it and confirmed it.